index.html

```diff
@@ -0,0 +1,99 @@
+        <a name="feature"></a>
+        <div class="boxes">
+            <h3>PORTFOLIO</h3>
+            <div class="projectBox project1">
+                <h4>PROJECT 1</h4>
+            </div>
+            <div class="projectBox project2">
+                <h4>PROJECT 2</h4>
+            </div>
+            <div class="projectBox project3">
+                <h4>PROJECT 3</h4>
+                <p>UNDER CONSTRUCTION</p>
+            </div>
+            <div class="projectBox project4">
+                <h4>PROJECT 4</h4>
+                <p>UNDER CONSTRUCTION</p>
+            </div>
+            <div class="projectBox project5">
+                <h4>PROJECT 5</h4>
+                <p>UNDER CONSTRUCTION</p>
+            </div>
+            <div class="projectBox project6">
+                <h4>PROJECT 6</h4>
+                <p>UNDER CONSTRUCTION</p>
+            </div>
+        </div>
+    </section>
+    <section class="aboutMe">
+        <a name="about"></a>
+        <div class="container">
+            <h2>ABOUT ME</h2>
+            <p>My name is Richard and I am aspiring to become a full stack developer. I am currently studying at the Mayden Academy in Bath and have been recently certified as a Scrum Master.</p>
+            <p>In addition to my growing web development skills, I have a Bachelor of Arts degree in Architecture and have been working as an Architectural Technician for the last 3 years. During that time I have acquired skills that are similar in nature to being a developer as well as soft skills that makes me an excellent addition to a team. In my free time, I enjoy photography, cooking and creating digital artworks using architectural software.</p>
+        </div>
+    </section>
+    <footer class="footer">
+        <a name="footer"></a>
+        <section class="contact">
+                <h2>CONTACT ME</h2>
+                <p>02637 615387</p>
+                <p>emailme@email.com</p>
+        </section>
+        <div class="container">
+            <button><a href="#top">Back to Top</a></button>
+        </div>
+        <div class="container">
+            <div class="logoContainer">
+                <a href="https://github.com/rrrichard" target="_blank">
+                    <img src="images/github.svg" class="invert" alt="github logo">
+                </a>
+                <a href="https://uk.linkedin.com/in/richard-mc-rabulan-ba7a94170" target="_blank">
+                    <img src="images/linkedin.svg" class="invert" alt="linkedin logo">
+                </a>
+                <a href="https://www.codewars.com/users/richarduuu" target="_blank">
+                    <img src="images/codewars.svg" class="invert" alt="codewars logo">
+                </a>
+                <a href="http://www.google.com" target="_blank">
+                    <img src="images/instagram.svg" class="invert" alt="instagram logo">
+                </a>
+            </div>
+        </div>
+        <div class="container">
```

Comment:
Same thing here, I think you could avoid this container. Bare in mind it aligns everything so you might need to change things in the css.

Reply:
I need it so it aligns with the texts on top